.github/workflows/post_peek_screenshot.yml

```
@@ -20,22 +20,22 @@ jobs:
         run: echo $PEEK_STATUS
 
       - name: Download workflow artifact
-        uses: dawidd6/action-download-artifact@v2.11.0
+        uses: dawidd6/action-download-artifact@v2.26.1
```

There is a recently update to that [**action**](https://github.com/dawidd6/action-download-artifact). 
```suggestion
        uses: dawidd6/action-download-artifact@v2.27.0
```